src/components/Textfield/Textfield.tsx

```diff
@@ -110,15 +111,7 @@ export default function TextField(props: Props): JSX.Element {
     <div className={`textfield ${className}`}>
       <label htmlFor={id} className='textfield-label'>
         {label}
-        {tooltipText !== undefined && (
-          <div className='textfield-label--icon-tooltip-container'>
-            <Tooltip title={tooltipText} placement='top' arrow={true}>
-              <span>
-                <Icon name='info' className='textfield-label--icon-tooltip'/>
-              </span>
-            </Tooltip>
-          </div>
-        )}
+        {tooltipTitle && <IconTooltip title={tooltipTitle} />}
```

Comment:
this should have `placement={'top'}` to match previous implementation

Reply:
Nice catch, thanks! 🙌

Reply:
sigh, sorry for coming back to this again. Looks like we'll need to add a string? tooltip placement prop to TextField (defaulted to 'top'). The figma shows all the added tooltips as 'top-start', but we'll want to have existing tooltips 'top' I guess.